applications/crossbar/src/modules/cb_user_auth.erl

```diff
@@ -445,11 +451,15 @@ maybe_load_user_doc_via_reset_id(Context) ->
     AccountDb = reset_id(ResetId),
     lager:debug("looking up password reset doc: ~s", [ResetId]),
     case kz_datamgr:open_cache_doc(AccountDb, ResetId) of
-        {'ok', [ResetIdDoc]} ->
+        {'ok', ResetIdDoc} ->
             lager:debug("found password reset doc"),
-            _ = kz_datamgr:del_doc(AccountDb, ResetIdDoc),
+            UserId = kz_json:get_value(<<"pvt_userid">>, ResetIdDoc),
+            UserDoc = crossbar_doc:load(UserId, Context, ?TYPE_CHECK_OPTION(kzd_user:type())),
```

Comment:
@fenollp i think i merged too fast.
`UserDoc = crossbar_doc:load(UserId, Context, ?TYPE_CHECK_OPTION(kzd_user:type())),`
the result of  `crossbar_doc:load ` is context() not doc